app/models/workshop.rb

```diff
@@ -2,6 +2,8 @@ class Workshop < ActiveRecord::Base
   include Invitable
   include Listable
 
+  attr_accessor :local_date, :local_time, :rsvp_open_local_date, :rsvp_open_local_time
```

Comment:
These temporary attributes hold the values from the date and time pickers in the form. We don't need to save them to the database